<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text File Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="fileContent">
      <property name="placeholderText">
       <string>File content will be displayed here...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_action">
        <property name="text">
         <string>Action:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="actionComboBox">
        <item>
         <property name="text">
          <string>Find</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Edit</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_search">
        <property name="text">
         <string>Search Term:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_add">
        <property name="text">
         <string>Line to Add:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="addInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_edit">
        <property name="text">
         <string>New Content:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editInput"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="actionButton">
      <property name="text">
       <string>Execute Action</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="resultOutput">
      <property name="placeholderText">
       <string>Results will be shown here...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
